images/grafana/config/latest.apko.yaml

```diff
@@ -0,0 +1,75 @@
+contents:
+  packages:
+    - busybox
+    - grafana
```

Comment:
Should we remove grafana here since it's included in "extra_packages"?

Reply:
For sure :) pushed up the suggested change